frontend/smart-kickers-game/src/components/Game/GameStatistics.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './GameStatistics.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Button } from '../Button/Button.js';
+
+function GameStatistics({ finalScores, handleEndGame, handleResetGame }) {
+  function handleNewGame() {
+    handleEndGame();
+  }
```

Comment:
The function `handleNewGame` wouldn't be necessary if you changed the name of the `handleEndGame` prop to e.g. `handleNewGame` and call it below.

Reply:
Done in a slightly different way. Reseting and ending game methods are now invoked after pressing endGame button.